### PR TITLE
ADEN-2967 Increase amount of Krux segments to 50

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

**Comment by jacekjursza** [comment]
LGTM

File: resources/wikia/modules/spec/krux.spec.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('Krux module', function () {
...
-			kruxSegments27 = ["l7lznzoty", "l4ml7tc6y", "l4w5i2lte", "l555eyz3i", "l5g2q8ndp", "l5h9g8s81", "l5pog7454", "l60oj8o6a", "l65e7q72q", "l6dwvwk4q", "l6s4rzsar", "l6sy2oz2g", "m2q9mod98", "m3049zgxw", "m68e86ids", "mab53nyfo", "mab7cpvql", "mdfzhvp3x", "mf20tfg50", "mjdpm83vl", "mkcdphvyq", "moz384t50", "mt6nf08we", "mueu17q3u", "myuf1vmwd", "n42nikugn", "n4mr5r49z"];
+			kruxSegments27 = ["l7lznzoty", "l4ml7tc6y", "l4w5i2lte", "l555eyz3i", "l5g2q8ndp", "l5h9g8s81", "l5pog7454", "l60oj8o6a", "l65e7q72q", "l6dwvwk4q", "l6s4rzsar", "l6sy2oz2g", "m2q9mod98", "m3049zgxw", "m68e86ids", "mab53nyfo", "mab7cpvql", "mdfzhvp3x", "mf20tfg50", "mjdpm83vl", "mkcdphvyq", "moz384t50", "mt6nf08we", "mueu17q3u", "myuf1vmwd", "n42nikugn", "n4mr5r49z", "n6lut96hn", "nal53baeb", "ne8px8c8b", "nm90py68x", "ntvki36t7", "o4r0zitna", "o854qerhj", "o854virtj", "o86oeigis", "o86ohot8d", "o95199u9p", "ocr1te1tc", "ocr52415y", "ocry7a4xg", "odm1prs9l", "oefi4m6j2", "ofbnn8fc5", "ofbnrmvhq", "ofbnwbwaj", "ofe048xdm", "ojahmr70e", "ot2f7awir", "ot2gg03he"];
```
Comment (gabrys) [inline resources/wikia/modules/spec/krux.spec.js]: Wrong name!

Comment (drets) [inline resources/wikia/modules/spec/krux.spec.js]: :+1:

**Comment by nandy-andy** [comment]
LGMT :+1:

**Pull request merged**